code/game/area/areas.dm

```diff
@@ -23,6 +23,7 @@
 									'sound/ambience/ambigen8.ogg','sound/ambience/ambigen9.ogg',\
 									'sound/ambience/ambigen10.ogg','sound/ambience/ambigen11.ogg',\
 									'sound/ambience/ambigen12.ogg','sound/ambience/ambigen14.ogg')
+	var/can_be_dirty = TRUE //Set this to FALSE to prevent an area from being made dirty at all
```

Comment:
`&` just checks if left has any of the 1s that right has.
````
10100101
&
00000100
=
00000100

10101001
&
00000100
=
00000000
````
I should also note that every var assigned a value adds 12 bytes of memory to the object. and every var added slows down all var accesses for those objects slightly.